src/client.ts

```diff
@@ -16,13 +16,8 @@
 
 import * as path from 'path';
 
-import { Storage, StorageOptions, PredefinedAcl } from '@google-cloud/storage';
-import {
-  randomFilepath,
-  inParallel,
-  toPlatformPath,
-  toPosixPath,
-} from '@google-github-actions/actions-utils';
+import { Storage, StorageOptions, PredefinedAcl, UploadOptions } from '@google-cloud/storage';
+import { inParallel, toPlatformPath, toPosixPath } from '@google-github-actions/actions-utils';
```

Comment:
Should we use actions/core instead since these were merged upstream??

Reply:
We already have the dependency on actions-utils. It could be a good followup, but it doesn't save us much.